addons/l10n_cn/models/account_move.py

```diff
@@ -21,15 +19,11 @@ def _check_fapiao(self):
             if record.fapiao and (len(record.fapiao) != 8 or not record.fapiao.isdecimal()):
                 raise ValidationError(_("Fapiao number is an 8-digit number. Please enter a correct one."))
 
-    @api.model
-    def check_cn2an(self):
-        return an2cn
-
     @api.model
     def _convert_to_amount_in_word(self, number):
         """Convert number to ``amount in words`` for Chinese financial usage."""
-        if not self.check_cn2an():
-            return None
+        if importlib.util.find_spec('an2cn') is None:
+            raise UserError(_('an2cn library is not installed'))
```

Comment:
Context? I'm not completely against changing that, but why do you need to change it?
It's best to keep these types of changes for the master branch.
Are you sure we want to raise in this case instead of not returning?

Reply:
I was waiting for the test to be all green to ping you for this change lol
you mentioned you want some change for above so i found a way to do the checking
Ref: https://github.com/odoo/odoo/blob/17.0/addons/web/controllers/vcard.py#L19

But yea I also think it would be wise to remain unchanged for the whole account.move model

Reply:
What I meant is to not add the line unless we need to change anything nearby, there is no need to add changes just for that ^^
IMO you should leave this file untouched